client/src/pages/ProfileCompleted.js

```diff
@@ -39,12 +52,18 @@ const ProfileCompleted = ({ user }) => {
             tertiary={true}
             id={GTM.user.profilePrefix + GTM.profile.continuePosting}
           >
-            Continue posting
+            {sessionStorage.getItem("createPostAttemptLoggedOut")
+              ? "Continue posting"
+              : "View Help Board"}
           </StyledButton>
         </Link>
       </ProfileCompletedButtonsWrapper>
     </ProfileCompletedWrapper>
   );
 };
 
-export default ProfileCompleted;
+const mapDispatchToProps = {
+  refetchUser,
+};
+
+export default connect(null, mapDispatchToProps)(withRouter(ProfileCompleted));
```

Comment:
As per above - remove `withRouter`

```suggestion
export default connect(null, mapDispatchToProps)(ProfileCompleted);
```